sandbox/deploy.sh

```diff
@@ -10,8 +10,16 @@ if [ "$1"x = "x" ]; then
     ns_num=1;
 elif [ "$1x" == "raftx" ]; then
     ns_num=3
-elif [ "$1x" == "master_slave" ]; then
+elif [ "$1" == "master_slave" ]; then
```

Comment:
I think each one is ok!  when we use `""`, it is OK to use "$1"even when $1 == NULL.
Traditionally, we can use `[ x$1 == x ]` or `[ "$1" == "" ]`, but combining them together seems werid.
Of course, it is my own idea.